charts/retool/templates/deployment_backend.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ template "retool.fullname" . }}
+  name: {{ template "retool.fullname" . }}-main-backend
```

Comment:
added to differentiate against workflows-backend

Reply:
it does seem fine - I upgraded my balloon instance. it just spun up the new main backends with this new name 

![image](https://github.com/tryretool/retool-helm/assets/2534903/2395aff7-0971-4403-96fc-54c784421d7f)

Reply:
ok awesome, thanks

Reply:
going to actually revert this - not too much benefit, and will be less disruptive k8s change (and will treat like a regular rotation of services, vs. stop the old, start the new)